packages/core/src/client/index.ts

```diff
@@ -259,7 +261,7 @@ export class Web3InboxClient {
     const domainToSearch = domain ?? this.domain;
 
     console.log(
```

Comment:
Logs help a lot while debugging but could be annoying for the production apps for our users. Wdyt about removing them?

Reply:
oh absolutely - i plan on removing them before merging `refactor/hooks-v1`. This just helps us debugging now

Reply:
Awesome, cool then